iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift

```diff
@@ -8,30 +8,30 @@ class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSo
 
     let LOCATION_LAKE_WASHINGTON = MSGeopoint(latitude: 47.609466, longitude: -122.265185)
     let customMapStyleString = """
-        {
-            "version": "1.0",
-            "settings": {
-                "landColor": "#FFFFFF",
-                "spaceColor": "#000000"
+    {
```

Comment:
I think original indentation actually helped with readability here, can you please keep it the way it was?